server/public/html/index.html

```diff
@@ -1,43 +1,239 @@
+										 	if (data.error) {
+												alert(data.error);
+											}
+											location.reload(true);
+									 }
+						});
+					} else {
+						alert("No file selected!");
+					}
+					return false;
+			});
+		});
+
+	</script>
 </head>
 
-<body style="display: block;">
-  <div id="welcome-content">
+<body>
+	<nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top">
+		<a class="navbar-brand" href="/">
+		  <img src="../logo.svg" class="ud-logo" alt="Annotatrix">
+		</a>
+	  <!-- <a class="navbar-brand" href="#">UD Annotatrix</a> -->
 
-		<h1 id="welcome-banner">Welcome to UD Annotatrix</h1>
+	  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+		<span class="navbar-toggler-icon"></span>
+	  </button>
 
-		<form method="post" enctype="multipart/form-data" action="./upload">
-      <input type="file" name="file" />
-      <input type="submit" value="Upload to server" />
-		</form>
+	  <div class="collapse navbar-collapse justify-content-center" id="navbarSupportedContent">
+		<ul class="navbar-nav">
 
-		<form method="post" action="./upload">
-			<input id="load-from-url" name="url" />
-			<input type="submit" value="Load from url" />
-		</form>
+		  <!-- <li class="nav-item active">
+			<a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+		  </li> -->
+		  <li class="nav-item">
+				<div class="input-group mb-3 form-inline my-2 my-lg-0">
+					<a class="btn btn-default btn-outline-success" href="./help.html" target="_blank">Help</a>
+			</div>
+		  </li>
+		  <li class="nav-item">
+				<div class="input-group mb-3 form-inline my-2 my-lg-0">
+					<a class="btn btn-default btn-outline-secondary" href="./annotatrix.html" target="_blank">Annotate!</a>
+				</div>
+		  </li>
+				<li class="nav-item">
+					<div class="input-group mb-3 form-inline my-2 my-lg-0">
+						<input type="text" id="gitlink" class="form-control" placeholder="URL">
+						<div class="input-group-append">
+							<button class="btn btn-default btn-outline-secondary" id="webload" type="submit">Load</button>
+						</div>
+					</div>
 
-		<div id="links">
-			<strong>Links:</strong>
-			<a id="to_annotatrix" href="./annotatrix.html">
-				Online annotator
-			</a>
-			<a id="to_help" href="./help.html">
-				Help
+					<!-- <form class="form-inline my-2 my-lg-0">
+					<input class="form-control mr-sm-2" type="search" placeholder="URL" aria-label="Search">
+					<button class="btn btn-outline-success my-2 my-sm-0" type="submit">Load</button>
+				</form> -->
+			 </li>
+		  <li class="nav-item">
+				<div class="input-group">
+	              <div class="input-group-prepend">
+	                <!-- <span class="input-group-text" id="inputGroupFileAddon01">Upload</span> -->
+									 <button id="upload-to-server" class="btn btn-outline-secondary" type="button">Upload</button>
```

Comment:
Why the extra indent?

Reply:
I will reindent it with Atom Beautify plugin.